scripts/build-maven-compile.bash

```diff
@@ -47,7 +47,14 @@ if [ -z "${M2_LOCATION}" ]; then
 	M2_LOCATION="./.m2"
 fi
 
-./mvnw compile --no-transfer-progress -Dmaven.repo.local="${M2_LOCATION}" # TODO: Maybe this should run offline?
-./mvnw process-test-resources --no-transfer-progress -Dmaven.repo.local="${M2_LOCATION}" --offline
-./mvnw test-compile --no-transfer-progress -Dmaven.repo.local="${M2_LOCATION}"
-./mvnw process-test-classes --no-transfer-progress -Dmaven.repo.local="${M2_LOCATION}" # TODO: Maybe this should be run offline?
+MAVEN_OPTIONS=" -Dmaven.repo.local=${M2_LOCATION} --no-transfer-progress "
+
+#./mvnw compile --no-transfer-progress -Dmaven.repo.local="${M2_LOCATION}" # TODO: Maybe this should run offline?
+#./mvnw process-test-resources --no-transfer-progress -Dmaven.repo.local="${M2_LOCATION}" --offline
+#./mvnw test-compile --no-transfer-progress -Dmaven.repo.local="${M2_LOCATION}"
+#./mvnw process-test-classes --no-transfer-progress -Dmaven.repo.local="${M2_LOCATION}" # TODO: Maybe this should be run offline?
```

Comment:
Is there a reason for this commented out stuff?

Reply:
Removed now,